py/desisim/scripts/quickspectra.py

```diff
@@ -22,9 +22,47 @@
 from desispec.spectra import Spectra
 from desispec.resolution import Resolution
 
+def _fft_gaussian_smooth(array, sigmapix, pad_size=10):
+    iwavesize, nspec = array.shape
+    # Pad the input array to get rid of annoying edge effects
+    # Pad values are set to the edge value
+    arrsize    = iwavesize+2*pad_size
+    padded_arr = np.empty((arrsize, nspec))
+    padded_arr[:pad_size, :] = array[0, :]
+    padded_arr[iwavesize+pad_size:, :] = array[-1, :]
+    padded_arr[pad_size:iwavesize+pad_size, :] = array
+
+    kvals        = np.fft.rfftfreq(arrsize)
+    kernel_k     = np.exp(-(kvals*sigmapix)**2/2.)
+    snumsource_k = np.fft.rfft(padded_arr, axis=0)*kernel_k[:, None]
+
+    return np.fft.irfft(snumsource_k, n=arrsize, axis=0)[pad_size:-pad_size]
```

Comment:
I haven't checked the pythonology of arrays and FFT conventions here, but I guess you have checked this.

Reply:
I can switch to fftconvolve. Any tricks for edge effects?
Why is this implementation not normalized though? kernel_k[0]=1, so sum of kernel_x=1. That's what I thought at least.

Reply:
(my bad, I was thinking about a kernel in configuration space, as it is the case in the argument of fftconvolve)